boards/BMS/Inc/Io/Io_LTC6813/Io_SharedLTC6813.h

```diff
@@ -45,21 +45,21 @@ enum RegGroupByteFormat
  * @param tx_data The command or register group to pack PEC15 bytes
  * @param size The size of the command or register group to transmit
  */
```

Comment:
rename to `num_bytes` and update comment

maybe mention that it's the number of bytes to transmit without the PEC15 since from first glance I would think it's the size of the tx_data buffer

Reply:
ah yeah nice catch, looks like some comments i forgot to update